Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(jenkinsVersions: [null, '2.32.3', '2.47'])
+buildPlugin(jenkinsVersions: [null, '2.60.3', '2.73.1'])
```

Comment:
2.7.x–2.47 uses one version of Groovy, 2.48–2.60.x another, 2.61+ yet another. At least we are dropping 1.x!